pkg/emb6/Makefile.include

```diff
@@ -4,6 +4,15 @@ EMB6_CONTRIB := $(RIOTBASE)/pkg/emb6/contrib
 
 INCLUDES += -I$(PKG_BUILDDIR)/target
 INCLUDES += -I$(RIOTBASE)/pkg/emb6/include
+INCLUDES += -I$(EMB6_DIR)/emb6
```

Comment:
are these globally needed?

Reply:
Yes, either the application or `auto_init` (if we ever do that for `emb6`) need the emb6 headers to bootstrap the stack.